### PR TITLE
Fix backdrop not being shown while searching resources on mobile

### LINKED ISSUE #8007: Search Bar Backdrop in ActionBarSearchBox is not visible
## Observed behavior

There is code for a backdrop that should be shown when the search box is open on small screens. But it doesn't look like it is ever visible. Is this just cruft?

https://github.com/learningequality/kolibri/blob/a9d1f4b579804d587b90580a0a9eeaf1a7dee955/kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue#L29

In Chrome devtools, it looks like it's  a zero-height div in the ActionBar.

CleanShot 2021-04-14 at 13 39 18

## Expected behavior

Backdrop is shown, or otherwise it's removed in the code.

### PR DESCRIPTION
**Pull request by Priyaraj17**
## Summary

The search bar backdrop in ActionSearchBox was not visible. We just have to test which height would have been perfect. Earlier, the backdrop was acting as zero-height div element.

…

## References

Issue: #8007
…

## Reviewer guidance

I have tested for all the available screen sizes using the chrome dev-tools and I am also providing the screenshots.

## Screenshots
WhatsApp Image 2021-05-10 at 4 19 01 PM (1)
WhatsApp Image 2021-05-10 at 4 19 01 PM (2)
WhatsApp Image 2021-05-10 at 4 19 01 PM

…

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by Priyaraj17** [comment]
@rtibbles @jonboiser is this PR okay?

**Comment by jonboiser** [comment]
Could you squash your two commits into 1?

Also, like you mentioned, the backdrop does not prevent the user from scrolling. You will need to add some code like this that disables scrolling when the backdrop is visible, then re-enables it when the backdrop is removed.

https://github.com/learningequality/kolibri-design-system/blob/v0.2.x/lib/KModal.vue#L216

**Comment by Priyaraj17** [comment]
Sure, I will work on it...Thanks!

**Comment by Priyaraj17** [comment]
Hey @jonboiser I have updated the changes...Hope it's okay. Thanks!

File: kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue [hunk 1]
```diff
@@ -76,9 +76,11 @@
       openDropdownSearchBox() {
         this.searchBoxIsOpen = true;
         this.focusOnSearchBox();
+        window.document.documentElement.style['overflow'] = 'hidden';
       },
       closeDropdownSearchBox() {
         this.searchBoxIsOpen = false;
+        window.document.documentElement.style['overflow'] = 'visible';
```
Comment (jonboiser) [inline kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue]:
```suggestion
        window.document.documentElement.style['overflow'] = '';
```

File: kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue [hunk 2]
```diff
@@ -124,11 +126,12 @@
 
   .search-box-dropdown-backdrop {
     position: fixed;
-    top: 111px;
+    top: 116px;
     right: 0;
     bottom: 0;
     left: 0;
     z-index: 4;
+    height: 90vh;
```
Comment (jonboiser) [inline kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue]:
```suggestion
    height: 100vh;
```

File: kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue [hunk 3]
```diff
@@ -124,11 +126,12 @@
 
   .search-box-dropdown-backdrop {
     position: fixed;
-    top: 111px;
+    top: 116px;
```
Comment (jonboiser) [inline kolibri/plugins/learn/assets/src/views/ActionBarSearchBox.vue]:
```suggestion
    top: 117px; // 66px (search bar) + 55px (dropdown)
```

**Review by jonboiser — CHANGES_REQUESTED** [review]

**Review by jonboiser — APPROVED** [review]
Looks good, thanks!

**Pull request merged**